tests/common/plugins/sanity_check/checks.py

```diff
@@ -100,41 +100,58 @@ def check_interfaces(dut):
 def check_bgp_status(dut):
 
     def _check_bgp_status_helper():
-        neighbors.clear()
-        neighbors.update(dut.get_bgp_neighbors())
-        if neighbors:
-            down_neighbors = [k for k, v in neighbors.items()
-                              if v['state'] != 'established']
-            if down_neighbors:
-                check_result['down_neighbors'] = down_neighbors
-                check_result['failed'] = True
+        asic_check_results = []
+        bgp_facts = dut.bgp_facts(asic_index='all')
+        for asic_index, a_asic_facts in enumerate(bgp_facts):
+            a_asic_result = False
+            a_asic_neighbors = a_asic_facts['ansible_facts']['bgp_neighbors']
```

Comment:
Can we add this to the function dut.get_bgp_neighbors(). It might be useful in other tests as well.

Reply:
get_bgp_neighbors() is in SonicHost. In our case, dut is an instance of MultiAsicSonicHost. SonicHost doesn't have a reference to MultiAsicSonicHost. And we probably don't want methods like this in MultiAsicSonicHost - as then we would have to replicate all get_* methods in SonicHost.